content/_data/changelogs/weekly.yml

```diff
@@ -19574,6 +19574,70 @@
   # pull: 7754 (PR title: Update dependency stylelint-config-standard to v31)
   # pull: 7755 (PR title: Update Yarn to v3.5.0)
 
+  - version: '2.397'
+    date: 2023-03-28
+    banner: >
+      A <strong>new GPG signing key</strong> is used for the Jenkins weekly package repositories.
+      Follow the instructions in the <a href="/blog/2023/03/27/repository-signing-keys-changing/">Linux repository signing blog post</a> to install the new public key on your computer.
+    changes:
+      - type: rfe
+        category: rfe
+        pull: 7767
+        authors:
+          - mustafau
+        pr_title: Add missing Turkish translations for plugins management page
+        message: |-
+          Add missing Turkish translations for plugins management page.
+      - type: rfe
+        category: rfe
+        pull: 7762
+        authors:
+          - mustafau
+        pr_title: Add missing Turkish translations for tools management page
+        message: |-
+          Add missing Turkish translations for tools management page.
+      - type: bug
+        category: bug
+        pull: 7689
+        issue: 70730
+        authors:
+          - timja
+        pr_title: JENKINS-70730 Don't remove id inside symbol
+        message: |-
+          Don't remove id inside symbol.
+      - type: bug
+        category: bug
+        pull: 7735
+        issue: 70809
+        authors:
+          - mamh2021
+        pr_title: '[JENKINS-70809] - Fix "delete build" button overflow the side-panel'
+        message: |-
+          Fix "delete build" button text overflow.
+      - type: rfe
+        category: developer
+        pull: 6664
+        authors:
+          - dependabot[bot]
+        pr_title: Bump spring-framework-bom from 5.3.25 to 5.3.26
+        references:
+          - url: https://github.com/spring-projects/spring-framework/releases/tag/v5.3.26
+            title: Spring framework BOM 5.3.26 release notes
+          - pull: 7760
+        message: |-
+          Bump spring-framework-bom from 5.3.25 to 5.3.26.
```

Comment:
`spring-framework-bom` is not an English word, so it should either be placed in backticks or replaced with a valid English sentence. One example of a valid English sentence would be:

> Upgrade Spring Framework from 5.3.25 to 5.3.26.

CC @kmartens27